### PR TITLE
Led control

### PR DESCRIPTION
**Pull request by Bankst**

### CODE REVIEW

File: photon-server/src/main/java/org/photonvision/common/configuration/ConfigManager.java [hunk 1]
```diff
@@ -201,6 +201,7 @@ public void saveToDisk() {
 
             for (var pipe : camConfig.pipelineSettings) {
                 var pipePath = Path.of(subdir.toString(), "pipelines", pipe.pipelineNickname + ".json");
+                logger.debug("Saving " + camConfig.pipelineSettings.size() + " pipes!");
```
Comment (mcm001) [inline photon-server/src/main/java/org/photonvision/common/configuration/ConfigManager.java:204]: kil?

File: photon-server/src/main/java/org/photonvision/vision/pipeline/PipelineProfiler.java [hunk 2]
```diff
@@ -95,6 +95,6 @@ protected static String getReflectiveProfileString(long[] nanos) {
     }
 
     public static void printReflectiveProfile(long[] nanos) {
-        reflectiveLogger.trace(() -> getReflectiveProfileString(nanos));
+        //        reflectiveLogger.trace(() -> getReflectiveProfileString(nanos));
```
Comment (mcm001) [inline photon-server/src/main/java/org/photonvision/vision/pipeline/PipelineProfiler.java:98]: heh

**Review by mcm001 — APPROVED** [review]

**Comment by Bankst** [comment]
[Vision LED control](https://trello.com/c/pUbe7HAo/101-vision-led-control)

**Comment by Bankst** [comment]
[Cleanup hardware management stuff](https://trello.com/c/a5N2uOkR/93-cleanup-hardware-management-stuff)

**Pull request merged**